reactor-netty-http/src/main/java/reactor/netty/http/server/MicrometerHttpServerMetricsRecorder.java

```diff
@@ -162,4 +187,20 @@ public void recordConnectTime(SocketAddress remoteAddress, Duration time, String
 	public void recordResolveAddressTime(SocketAddress remoteAddress, Duration time, String status) {
 		throw new UnsupportedOperationException();
 	}
+
+	private LongAdder getServerConnectionAdder(String uri, String method) {
+		MeterKey meterKey = new MeterKey(uri, null, method, null);
+		LongAdder adder = activeConnectionsCache.get(meterKey);
+		adder = adder != null ? adder : activeConnectionsCache.computeIfAbsent(meterKey,
+				key -> {
+					Gauge gauge = filter(Gauge.builder(reactor.netty.Metrics.HTTP_SERVER_PREFIX + CONNECTIONS_ACTIVE,
+							activeConnectionsAdder, LongAdder::longValue)
+							.tags(URI, uri, METHOD, method)
```

Comment:
the tags would become:
`.tags(URI, PROTOCOL_VALUE_HTTP, LOCAL_ADDRESS, localAddress)`

Reply:
NB: amend the tests accordingly